app/controllers/users/invitations_controller.rb

```diff
@@ -185,12 +198,18 @@ def check_captcha_for_invite
     def check_invite_users_permission
       @user = current_user
       @emails = params[:emails]&.map(&:downcase)
-      @team = Team.find_by_id(params['teamId'])
+
+      if params['teamId']
+        @teams = [Team.find(params['teamId'])]
```

Comment:
may be it's better to use 'where' to get ActiveRecord::Relation?

Reply:
agreed